[
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "recursion exception depth 5",
          "type": "System.Exception",
          "stacktrace": [
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware+<Invoke><scrubbed>",
              "lineNumber": 0
            },
            {
              "<runtime>": "<scrubbed>"
            }
          ]
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "recursion exception depth 5"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "originalDepth": {
            "type": "Int32",
            "value": "5"
          },
          "currentDepth": {
            "type": "Int32",
            "value": "5"
          },
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "exceptionReplayRecursionHelper",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": [
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  },
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "recursion exception depth 5",
          "type": "System.Exception",
          "stacktrace": [
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "<runtime>": "<scrubbed>"
            }
          ]
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "recursion exception depth 5"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "originalDepth": {
            "type": "Int32",
            "value": "5"
          },
          "currentDepth": {
            "type": "Int32",
            "value": "0"
          },
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "exceptionReplayRecursionHelper",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": [
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware+<Invoke><scrubbed>",
        "lineNumber": 0
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware.Invoke",
        "lineNumber": 0
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  },
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "recursion exception depth 5",
          "type": "System.Exception",
          "stacktrace": [
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "<runtime>": "<scrubbed>"
            }
          ]
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "recursion exception depth 5"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "originalDepth": {
            "type": "Int32",
            "value": "5"
          },
          "currentDepth": {
            "type": "Int32",
            "value": "1"
          },
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "exceptionReplayRecursionHelper",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": [
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware+<Invoke><scrubbed>",
        "lineNumber": 0
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware.Invoke",
        "lineNumber": 0
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  },
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "recursion exception depth 5",
          "type": "System.Exception",
          "stacktrace": [
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "<runtime>": "<scrubbed>"
            }
          ]
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "recursion exception depth 5"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "originalDepth": {
            "type": "Int32",
            "value": "5"
          },
          "currentDepth": {
            "type": "Int32",
            "value": "2"
          },
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "exceptionReplayRecursionHelper",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": [
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
        "fileName": "/app/ExceptionReplayController.cs",
        "lineNumber": 38
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware+<Invoke><scrubbed>",
        "lineNumber": 0
      },
      {
        "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware.Invoke",
        "lineNumber": 0
      },
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  },
  {
    "captures": {
      "return": {
        "staticFields": {
          "Empty": {
            "type": "EmptyResult",
            "value": "EmptyResult"
          }
        },
        "throwable": {
          "message": "recursion exception depth 5",
          "type": "System.Exception",
          "stacktrace": [
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "weblog.ExceptionReplayController.exceptionReplayRecursionHelper",
              "lineNumber": 0
            },
            {
              "function": "Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.BlockingMiddleware+<Invoke><scrubbed>",
              "lineNumber": 0
            },
            {
              "<runtime>": "<scrubbed>"
            }
          ]
        },
        "locals": {
          "@exception": {
            "type": "Exception",
            "value": "Exception",
            "fields": {
              "Message": {
                "type": "String",
                "value": "recursion exception depth 5"
              },
              "InnerException": {
                "type": "Exception",
                "isNull": "true"
              },
              "HelpLink": {
                "type": "String",
                "isNull": "true"
              },
              "Source": {
                "type": "String",
                "value": "app"
              },
              "HResult": {
                "type": "Int32",
                "value": "-2146233088"
              },
              "StackTrace": {
                "type": "String",
                "value": "<scrubbed>"
              }
            }
          }
        },
        "arguments": {
          "originalDepth": {
            "type": "Int32",
            "value": "5"
          },
          "currentDepth": {
            "type": "Int32",
            "value": "5"
          },
          "this": {
            "type": "ExceptionReplayController",
            "value": "ExceptionReplayController"
          }
        }
      }
    },
    "probe": {
      "id": "<scrubbed>",
      "version": 1,
      "location": {
        "method": "exceptionReplayRecursionHelper",
        "type": "weblog.ExceptionReplayController"
      }
    },
    "stack": [
      {
        "<runtime>": "<scrubbed>"
      }
    ],
    "id": "<scrubbed>",
    "timestamp": "<scrubbed>",
    "duration": "<scrubbed>",
    "language": "dotnet"
  }
]